scripts/project.js

```diff
@@ -1,44 +1,52 @@
-function Project(opts){
-  this.title = opts.title;
-  this.projectUrl = opts.projectUrl;
-  this.publishedOn = opts.publishedOn;
-  this.projectDescription = opts.projectDescription;
-}
+(function(module) {
 
-Project.all = [];
+  function Project(opts){
+    this.title = opts.title;
+    this.projectUrl = opts.projectUrl;
+    this.publishedOn = opts.publishedOn;
+    this.projectDescription = opts.projectDescription;
+  }
 
-Project.prototype.toHtml = function() {
-  var source = $('#project-template').html();
-  var template = Handlebars.compile(source);
-  return template(this);
-}
+  Project.all = [];
 
-Project.loadAll = function(rawData) {
-  rawData.forEach(function(ele) {
-    Project.all.push(new Project(ele));
-  })
-};
+  Project.prototype.toHtml = function() {
+    var source = $('#project-template').html();
+    var template = Handlebars.compile(source);
+    return template(this);
+  }
 
-Project.fetchAll = function() {
-  var storedEtag;
-  $.ajax({
-    type: "HEAD",
-    url: "data/portfolioProjects.json",
-    success: function(data, message, xhr) {
-      storedEtag = xhr.getResponseHeader("etag");
-      if (storedEtag === localStorage.etag && localStorage.rawData) {
-        Project.loadAll(JSON.parse(localStorage.rawData));
-        projectView.initIndexPage();
-      } else {
-        $.getJSON('data/portfolioProjects.json', function(data) {
-          console.log(data);
-          Project.loadAll(data);
-          localStorage.setItem("etag", JSON.stringify(storedEtag));
-          localStorage.setItem('rawData', JSON.stringify(data));
+  Project.loadAll = function(rawData) {
+    rawData.forEach(function(ele) {
+      Project.all.push(new Project(ele));
+    })
+  };
+
+  Project.fetchAll = function() {
+    var storedEtag;
+    $.ajax({
+      type: "HEAD",
+      url: "data/portfolioProjects.json",
+      success: function(data, message, xhr) {
+        storedEtag = xhr.getResponseHeader("etag");
+        if (storedEtag === localStorage.etag && localStorage.rawData) {
+          Project.loadAll(JSON.parse(localStorage.rawData));
+          FunFact.loadAll(JSON.parse(localStorage.funFacts))
           projectView.initIndexPage();
```

Comment:
Why is this in this file? And why is it different from the FunFact.loadAll() in funFacts.js?